packages/test/package.json

```diff
@@ -1,9 +1,4 @@
 {
   "name": "test-package",
-  "version": "1.0.0",
-  "main": "index.js",
-  "license": "MIT",
-  "exports": {
-    ".": "./index.js"
-  }
+  "version": "1.0.0"
```

Comment:
with workspaces, you really don't need anything in your package.json (no main or exports)